ui/apps/lists/src/components/lists-page.tsx

```diff
@@ -21,15 +21,16 @@ const ListsPage: React.FC<unknown> = () => {
 
   const bookmarkDelete = null;
 
+  const { isLoggedIn } = useLoggedIn();
+
   const profileDataReq = useGetMyProfileQuery(null, {
```

Comment:
We can use `loggedProfileData` from `useLoggedIn` hook instead of recomputing it here :)

Reply:
This component needs updating yes